### PR TITLE
Fixed spelling errors in status messages.

### PR DESCRIPTION
**Pull request by TylerAlsop**
There was a spelling and grammar error the status message included in 3 different files. The word "already" was spelled wrong and the word "exist" needed to be "exists" to be grammatically correct.

### CODE REVIEW

File: api/middleware/animal.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ module.exports = {
         }
 
         if (await animal_qry.first()) {
-          res.status(404).json({ message: 'Animal alreday exist' });
+          res.status(404).json({ message: 'Animal already exists.' });
```
Comment (tanveersaleem786) [inline api/middleware/animal.js:50]: Nice Catch

File: api/middleware/animalService.js [hunk 2]
```diff
@@ -102,7 +102,7 @@ module.exports = {
           .where('service_id', service_id);
 
         if (await service_qry.first()) {
-          res.status(404).json({ message: 'Animal Service alreday exist' });
+          res.status(404).json({ message: 'Animal Service already exists.' });
```
Comment (tanveersaleem786) [inline api/middleware/animalService.js:105]: Thanks for fixing

File: api/middleware/service.js [hunk 3]
```diff
@@ -44,7 +44,7 @@ module.exports = {
         }
 
         if (await service_qry.first()) {
-          res.status(404).json({ message: 'Service alreday exist' });
+          res.status(404).json({ message: 'Service already exists.' });
```
Comment (tanveersaleem786) [inline api/middleware/service.js:47]: Great Job

**Review by tanveersaleem786 — APPROVED** [review]
Great Job Tyler

File: api/middleware/animal.js (hunk 1 above)
Comment (tommycoleman87) [reply api/middleware/animal.js:50]: Solid catch

File: api/middleware/animalService.js (hunk 2 above)
Comment (tommycoleman87) [reply api/middleware/animalService.js:105]: awesome

File: api/middleware/service.js (hunk 3 above)
Comment (tommycoleman87) [reply api/middleware/service.js:47]: awesome!

**Review by tommycoleman87 — APPROVED** [review]
Easy peasy fixes!

File: api/middleware/animal.js (hunk 1 above)
Comment (MLucky518) [reply api/middleware/animal.js:50]: nice catch man!

**Pull request merged**